conans/test/functional/generators/cmake_find_package_test.py

```diff
@@ -485,6 +485,101 @@ def build(self):
                       "$<$<STREQUAL:$<TARGET_PROPERTY:TYPE>,EXECUTABLE>:>",
                       client.out)
 
+    def cpp_info_filename_test(self):
```

Comment:
Although this test is nos specific to components, I believe it will be better to move it here: https://github.com/conan-io/conan/blob/develop/conans/test/functional/generators/components/cmake_find_package_multi_test.py